sync_with_branch.sh

```diff
@@ -53,26 +59,31 @@ pick_latest_result() {
     rm -rf "results"
 }
 
-# For a given bennchmark, render JSON results as a PGF plot (TeX), then compile
-# it to PDF and further convert to SVG. The PDF may contain multiple pages
-# (lexer generator benchmarks have file size charts in addition to time charts).
-gen_bar_charts_for_benchmark() {
-    benchmark="$1"
-    j2pp_args="$2"
-    ( cd "src/benchmarks/submatch_$benchmark" \
-        && echo "Generating SVG for $benchmark benchmark..." \
-        && pick_latest_result \
-        && python3 ../../../build/json2pgfplot.py \
-            --variant $benchmark \
-            --font comicneue \
-            $j2pp_args \
-            results.json results.tex \
-        && pdflatex results.tex < /dev/null > results.log \
-        && pdf2svg results.pdf results_1.svg 1 \
-        && rm results.{aux,log,pdf,tex} \
-    )
-}
+# # For a given bennchmark, render JSON results as a PGF plot (TeX), then compile
```

Comment:
I can't make these lines work on my machine.
It fails on the first `pdflatex results.tex` command.

Reply:
I have fixed it. Turned out the `pdflatex` was silently failing because it needed some latex package that wasn't installed. I have fixed it by installing `texlive-full`